entsoe-util/src/main/java/com/powsybl/entsoe/util/MergedXnodeImpl.java

```diff
@@ -106,6 +134,50 @@ public MergedXnodeImpl setXnodeQ1(double xnodeQ1) {
         return this;
     }
 
+    @Override
+    public MergedXnodeImpl setB1dp(float b1dp) {
```

Comment:
According to your implementation, `b1dp` (and others) must always be defined. Do default values exist? (for example 0.5). If they do, set `b1dp` (and others) as such when the given float is `NaN`

Reply:
I was inspired by rdp and xdp to implement these ratios.
Should I put a default value?

Reply:
@geofjamg What do you think? Are `b1dp` and new associate attributes mandatory from now on?